app/src/main/java/com/github/h3lp3rs/h3lp/SettingsActivity.kt

```diff
@@ -0,0 +1,109 @@
+package com.github.h3lp3rs.h3lp
+
+import android.content.Intent
+import android.content.SharedPreferences
+import android.os.Build
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.CheckBox
+import android.widget.TextView
+import androidx.annotation.RequiresApi
+import androidx.appcompat.app.AlertDialog
+import com.github.h3lp3rs.h3lp.signin.GoogleSignInAdapter.getCreationDate
+import com.github.h3lp3rs.h3lp.signin.GoogleSignInAdapter.signOut
+import com.github.h3lp3rs.h3lp.signin.SignInActivity
+import com.github.h3lp3rs.h3lp.signin.SignInActivity.Companion.getUid
+import com.github.h3lp3rs.h3lp.storage.Storages
+import com.github.h3lp3rs.h3lp.storage.Storages.*
+import com.github.h3lp3rs.h3lp.storage.Storages.Companion.storageOf
+import org.apache.commons.collections4.iterators.SkippingIterator
+
+class SettingsActivity : AppCompatActivity() {
```

Comment:
You could run a reformat file since there's a few spaces missing here and there to be consistent with the rest of the files